src/test/kotlin/com/cultureamp/eventsourcing/AsyncEventProcessorMonitorIntegrationTest.kt

```diff
@@ -44,13 +45,15 @@ class AsyncEventProcessorMonitorIntegrationTest : DescribeSpec({
         transaction(db) {
             addLogger(StdOutSqlLogger)
             SchemaUtils.create(table)
+            SchemaUtils.create(eventsSequenceStatsTable)
```

Comment:
One of these days we need to figure out when this is necessary and when it's a random incantation that we do for good luck. Did anything fail before you added this?

Reply:
Hahaha nah in this case it definitely failed without this, saying the table couldn't be found